### PR TITLE
[+] IP Whitelist - Refresh IP Whitelist on liana when update on front

### PR DESCRIPTION
**Pull request by ArnaudValensi**
## Pull Request checklist:

- [ ] Write an explicit title for the Pull Request
- [ ] Write changes made in the CHANGELOG.md
- [ ] Test manually the implemented changes
- [ ] Review my own code (indentation, syntax, style, simplicity, readability)
- [ ] Wonder if you can improve the existing code

### CODE REVIEW

**Comment by ArnaudValensi** [comment]
[(3) IP Whitelist - Refresh IP Whitelist on liana when update on front](https://trello.com/c/7orF7lLq/2355-3-ip-whitelist-refresh-ip-whitelist-on-liana-when-update-on-front)

File: routes/ip-whitelist-rules.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+'use strict';
+var path = require('../services/path');
+var auth = require('../services/auth');
+var ipWhitelist = require('../services/ip-whitelist');
+
+module.exports = function (app, opts) {
```
Comment (arnaudbesnier) [inline routes/ip-whitelist-rules.js]: opts => options

File: routes/ip-whitelist-rules.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  function refresh(req, res) {
```
Comment (arnaudbesnier) [inline routes/ip-whitelist-rules.js]: req => request

Comment (arnaudbesnier) [reply routes/ip-whitelist-rules.js]: res => response 🙏

**Pull request merged**